src/templates/mkt_nav.html

```diff
@@ -15,6 +24,18 @@
     {{ nav_item(url('new'), _('New'), 'new') }}
     {{ nav_item(url('recommended'), _('Recommended'), 'recommended') }}
     {{ nav_item('#', _('Categories'), 'categories', 'categories') }}
+    <div class="mkt-nav-root--divider"></div>
```

Comment:
This is where `<mkt-nav>` being the sidebar and the appbar doesn't make sense. So on desktop we have an appbar across the top `<mkt-header>` and it has the rocket and the magnifying glass and we slap the `<mkt-nav>` on top of it. Then on mobile we don't slap the `<mkt-nav>` on top and show the hamburger instead. Now `<mkt-nav>` is the sidebar.